src/Vector3.h

```diff
@@ -121,11 +126,7 @@ class Vector3
 	{
 		// Perform a bitwise comparison of the contents - we want to know whether this object is exactly equal
 		// To perform EPS-based comparison, use the EqualsEps() function
-		return (
-			(memcmp(&x, &a_Rhs.x, sizeof(x)) == 0) &&
-			(memcmp(&y, &a_Rhs.y, sizeof(y)) == 0) &&
-			(memcmp(&z, &a_Rhs.z, sizeof(z)) == 0)
-		);
+		return !((x != a_Rhs.x) || (y != a_Rhs.y) || (z != a_Rhs.z));
```

Comment:
The types we're comparing ARE POD - they're `int`, `float` or `double`, based on the template param.
Positive and negative zero most likely won't compare well with `!=` either, since clang basically warns that it's using the bitwise comparison. I'm still not convinced.